rmf_human_detector/launch/human_detector_launch.py

```diff
@@ -0,0 +1,73 @@
+import launch
+from launch import LaunchDescription
+from launch.actions import IncludeLaunchDescription
+from launch_ros.actions import Node
+from launch.substitutions import EnvironmentVariable
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from ament_index_python.packages import get_package_share_directory
+
+import os
+
+def generate_launch_description():
+    # Bridge
+    bridge = Node(
+        package='ros_ign_bridge',
+        executable='parameter_bridge',
+        arguments=[
+            '/model/camera1/pose@tf2_msgs/msg/TFMessage@ignition.msgs.Pose_V',
+            '/world/sim_world/model/camera1/link/visual_link/sensor/camera/image@sensor_msgs/msg/Image@ignition.msgs.Image',
+            '/world/sim_world/model/camera1/link/visual_link/sensor/camera/camera_info@sensor_msgs/msg/CameraInfo@ignition.msgs.CameraInfo',
+            ],
+        output='screen',
+        remappings=[
+            ('/model/camera1/pose', '/camera1/pose'),
+            ('/world/sim_world/model/camera1/link/visual_link/sensor/camera/image', '/camera1/image_raw'),
+            ('/world/sim_world/model/camera1/link/visual_link/sensor/camera/camera_info', '/camera1/camera_info'),
+        ]
+    )
+    return LaunchDescription([
+        Node(
+           package='rmf_human_detector',
+           executable='human_detector',
+           parameters=[
+                {"camera_name": "camera1"},
+                {"camera_level": "L1"},
+                {"nn_filepath": "/home/osrc/rmf_ws/src/rmf/rmf_obstacle_detectors/rmf_human_detector/assets/yolov5s.onnx"},
```

Comment:
Never hardcode file paths. As per readme instructions, these weights and names should have been copied into `assets/` folder which should then get installed during `colcon build`.

You should then retrieve these paths programatically.

```
share_dir = get_package_share_directory("rmf_human_detector")
model_path = os.path.join(share_dir, "assets", "yolov5s.onnx")
labels_path = os.path.join(share_dir, "assets", "coco.names")

```